src/pygambit/game.pxi

```diff
@@ -905,23 +906,42 @@ class Game:
         KeyError
             If `node` is a string and no node in the game has that label.
         TypeError
-            If `node` is not a `Node` or a `str`
+            If `node` is not either a `Node` or a `str` or an `Iterable`.
         ValueError
             If `node` is an empty `str` or all spaces
         """
+        ReferenceSet = []
```

Comment:
As I mentioned before variable names must *always* be snake_case.  I please don't want to have to ask again!